book/modules/appendix3.tex

```diff
@@ -0,0 +1,476 @@
+Determinant is useful: it measures the volume distortion of a linear transformation,
+it helps us determine if a matrix is invertible of not, and it provides us with a
+simple formula for solving $A\vec x=\vec y$ when $A$ is invertible.
+
+Module \ref{determinant module} discusses an algorithm for evaluating determinants
+via row reduction. However, for $2\times 2$ and $3\times 3$ determinants, we
+have direct formulas. We will explore these formulas in this appendix.
+
+\Heading{Calculating \(2\times 2\) Determinants} For a $2\times 2$ matrix, once we
+know its four entries, we can calculate its determinant directly.
+\begin{theorem}[$2\times 2$ determinant formula]\index{Determinant!$2\times 2$ formula}
+	Let $M=\mat{a & b \\ c & d}$. Then
+	\[
+		\det(M)=ad-bc.
+	\]
+\end{theorem}
+\begin{proof}
+Let $\mathcal{T}_{M}$ be the matrix multiplication mapping induced by \(M\), namely \[\mathcal{T}_{M}:\mathbb{R}^2\rightarrow \mathbb{R}^2,\quad \text{ where }\mathcal{T}(\vec x)=M\vec x.\]
+
+If $\Set{\matc{a \\ c},\matc{b \\ d}}$ is linearly dependent, then $\dim \Range(\mathcal{T}_M)<2$, then $\mathcal {T}_{{M}}(C_2)$ is either a point or a one-dimensional object. In either case it has signed area $0$, so $\det(\mathcal{T}_{M})=0$. Thus, $\det(M)=0$. Now, the fact that $\Set{\matc{a \\ c},\matc{b \\ d}}$ is linearly dependent implies that there exists a scalar $\lambda$ such that $\matc{a \\ c}=\lambda\matc{b \\ d}$. Then $ad-bc=(\lambda b)d-b(\lambda c)=0$. Thus, $\det(M)=ad-bc$ in the case when column vectors of $M$ are linearly dependent.
```

Comment:
You can use `\Set*{}` for the enlarged curly brackets.

Reply:
`C_2` is the unit 2-cube. But it would be good to remind the reader of that.